.gitignore

```diff
@@ -1 +1,4 @@
 .DS_Store
+*.gem
+Gemfile.lock
+.package-lock.json
```

Comment:
Mmm, I'd get rid of this. Yes it should be ignored, but there's no reason why it should exist to begin with.